app/services/transaction_fee_processor.rb

```diff
@@ -23,8 +23,10 @@ def execute
   def add_transaction_fee(order)
     return if @users.include?(order.portfolio.user_id)
 
-    order.portfolio.portfolio_transactions.fees.create!(amount_cents: PortfolioTransaction::TRANSACTION_FEE_CENTS,
-                                                        reason: PortfolioTransaction::REASONS[:transaction_fee])
+    order.portfolio.portfolio_transactions.fees.create!(
+      amount_cents: PortfolioTransaction::TRANSACTION_FEE_CENTS,
+      reason: :transaction_fees
```

Comment:
I think it was incorrect in the original form 🤔 

We don't have `transaction_fees` reason. 

Fee is a transaction type